internal/util/sync/updatefile.go

```diff
@@ -53,7 +53,10 @@ func SetDependency(dependency kptfile.Dependency) error {
 	// add the dependency
 	if !found {
 		if dependency.Strategy == "" {
-			dependency.Strategy = string(update.FastForward)
+			dependency.Strategy = string(update.KResourceMerge)
+		} else if !update.ValidStrategy(dependency.Strategy) {
```

Comment:
(reviewing lines not changes)

Line 27: Consider adding utility for reading the Kptfile instead of doing low-level ioutil.ReadFile anytime we need to read the Kptfile

Reply:
Good catch. There are methods in-place but not used. Updated.